.github/pre-commit-config.yaml

```
@@ -79,10 +82,11 @@ repos:
       - id: mixed-line-ending
-        name: Check line nedings
+        name: Check line endings
         language: system
         types: [text]
         stages: [commit-msg, post-commit, manual]
         entry: mixed-line-ending
         args:
-          - --fix=lf
\ No newline at end of file
+          - --fix=lf
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)